open-reac/src/main/java/com/powsybl/openreac/parameters/output/OpenReacResult.java

```diff
@@ -97,8 +113,55 @@ public List<NetworkModification> getAllModifs() {
     }
 
     public void applyAllModifications(Network network) {
-        for (NetworkModification modif : getAllModifs()) {
+        for (NetworkModification modif : getAllNetworkModifications()) {
             modif.apply(network);
         }
+
+        // update target of ratio tap changers regulating voltage
+        network.getTwoWindingsTransformerStream()
```

Comment:
Sorry I didn't see that in the first place, but you have to take the transformer from the id of the generator modifications, same for shunts.

Reply:
Done!